BedrockServer.h

```diff
@@ -172,7 +172,8 @@ class BedrockServer : public SQLiteServer {
     static constexpr auto STATUS_HANDLING_COMMANDS = "GET /status/handlingCommands HTTP/1.1";
     static constexpr auto STATUS_PING              = "Ping";
     static constexpr auto STATUS_STATUS            = "Status";
-    static constexpr auto STATUS_WHITELIST         = "SetCommandWhitelist";
+    static constexpr auto STATUS_BLACKLIST         = "SetCommandBlacklist";
```

Comment:
Should this be `SetParallelCommandBlacklist`?  After all, we're not blacklisting commands entirely.